contrib/gitian-build.py

```diff
@@ -72,6 +72,8 @@ def build():
 
     if args.macos:
         print('\nCompiling ' + args.version + ' MacOS')
+        subprocess.check_call(['wget', '-N', '-P', 'inputs', 'https://bitcoincore.org/depends-sources/sdks/MacOSX10.11.sdk.tar.gz'])
```

Comment:
I'd wrap this in `if not os.path.isfile('gitian-builder/inputs/MacOSX10.11.sdk.tar.gz'):` to avoid re-downloading it over and over again.

Reply:
wget will skip the download with `-N` if file exists and headers now showing outdated version

Reply:
Ah, ok, cool 👍  Learning smth new here :)